docs/source/test_network.md

```diff
@@ -0,0 +1,594 @@
+  tests by the Fabric SDK. Refer to the [Node SDK](https://github.com/hyperledger/fabric-sdk-node)
+  repo for details on running these tests.
+
+- If you use the `createChannel` subcommand, `./network.sh` runs the
+  `createChannel.sh` script in the `scripts` folder to create a channel
+  using the supplied channel name. The script uses the `configtx.yaml` file to
+  create the channel creation transaction, as well as two anchor peer update
+  transactions. The script uses the peer cli to create the channel, join
+  ``peer0.org1.example.com`` and ``peer0.org2.example.com`` to the channel, and
+  make both of the peers anchor peers.
+
+- If you issue the `installCC` command, `./network.sh` runs the ``installCC.sh``
+  script to install the **fabcar** chaincode on both peers and then define then
+  chaincode on the channel. Once the chaincode definition is committed to the
+  channel, the peer cli initializes the chainocde using the `Init` and invokes
+  the chaincode to put initial data on the ledger.
+
+# Troubleshooting
+
+If you have any problems with the tutorial, review the following:
+
+-  You should always start your network fresh. You can use the following command
+   to remove artifacts, crypto material, containers, and chaincode images:
+   ```
+   ./network.sh down
+   ```
+   You **will** see errors if you do not remove old containers and images.
+
+-  If you see Docker errors, first check your Docker version ([Prerequisites](prereqs.html)),
+   and then try restarting your Docker process. Problems with Docker are
+   oftentimes not immediately recognizable. For example, you may see errors
+   that are the result of your node not being able to access the crypto material
+   mounted within a container.
+
+   If problems persist, you can remove your images and start from scratch:
+   ```
+   docker rm -f $(docker ps -aq)
+   docker rmi -f $(docker images -q)
+   ```
+
+-  If you see errors on your create, approve, commit, invoke or query commands,
+   make sure you have properly updated the channel name and chaincode name.
+   There are placeholder values in the supplied sample commands.
+
+-  If you see the error below:
+   ```
+   Error: Error endorsing chaincode: rpc error: code = 2 desc = Error installing chaincode code mycc:1.0(chaincode /var/hyperledger/production/chaincodes/mycc.1.0 exits)
+   ```
+
+   You likely have chaincode images (e.g. ``dev-peer1.org2.example.com-fabcar-1.0`` or
+   ``dev-peer0.org1.example.com-fabcar-1.0``) from prior runs. Remove them and try
+   again.
+   ```
+   docker rmi -f $(docker images | grep peer[0-9]-peer[0-9] | awk '{print $3}')
+   ```
+
+-  If you see something similar to the following:
+   ```
+   Error connecting: rpc error: code = 14 desc = grpc: RPC failed fast due to transport failure
+   Error: rpc error: code = 14 desc = grpc: RPC failed fast due to transport failure
+   ```
+
+   Make sure you are running your network against the "1.0.0" images that have
```

Comment:
is it 1.0.0??

Reply:
This error must have been in the docs for a while. current network does not even work with 1.0.0. Will remove